src/main/java/core/basesyntax/model/User.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.model;
+
+public class User {
+    private String name;
+
+    User(String name) {
+        this.name = checkName(name);
+    }
+
+    public String getName() {
+        return name;
+    }
```

Comment:
create setter
and maybe let's add `age` field as well

Reply:
done